ui/app/components/sidebar/nav/cluster.hbs

```diff
@@ -13,7 +13,7 @@
     @text="Secrets Engines"
     data-test-sidebar-nav-link="Secrets Engines"
   />
-  {{#if this.flags.showSecretsSync}}
+  {{#if this.showSecretsSync}}
```

Comment:
I think we still need to account for hvd clusters who won't have this on their license so we need to show it regardless.

Reply:
Added!